scripts/suse_setup_libvirt.sh

```diff
@@ -39,6 +39,7 @@ zypper -n install \
     vagrant-libvirt=0.7.0-bp154.1.101 \
```

Comment:
vagrant plugin install vagrant-libvirt doesn't work for me on suse. Instead I install it here ^.

Reply:
Interesting, I found on debian that despite doing `apt install vagrant-libvirt` that I still needed to do `vagrant plugin install`. Must be different on SUSE. Fixing.